qiskit_quantuminspire/qi_jobs.py

```diff
@@ -20,7 +44,7 @@ def __init__(
         self,
         run_input: Union[QuantumCircuit, List[QuantumCircuit]],
         backend: Union[Backend, None],
-        job_id: str,
+        job_id: Optional[str] = None,
```

Comment:
Also, for clarity we really should be renaming this to batch_job_id

Reply:
Changed name